### DISCUSSION TITLE
plot_beta + FamNames and Index in lattice + zoom

### DISCUSSION THREAD

**Discussion by simoneliuzzo** [atcollab/at#462] [Python AT]
matlab atplot has options to plot the magnet names and the index along the lattice. Those are very useful functions to visualize and use the lattice. 
the pyAT version of atplot (plot_beta) is unfortunately missing these two options.

**Comment by simoneliuzzo** [comment]
also the plot only a given s range seems not available in pyAT

**Comment by lfarv** [comment]
Hi @simoneliuzzo,
The range of interest for the plot can be selected with the `s_range` keyword.
But you are right concerning magnet names and indices. It's now on the todo list !
